sematic/db/migrate.py

```diff
@@ -159,7 +159,7 @@ def _update_version(conn: Connection, version: str, direction: MigrationDirectio
         else "DELETE FROM schema_migrations WHERE version = :version"
     )
 
-    conn.execute(text(statement), version=version)
+    conn.execute(text(statement), {"version": version})
```

Comment:
kwargs are no longer allowed for binding prepared statements. You have to pass a dict param.